src/frida/repl.py

```diff
@@ -423,6 +431,57 @@ def _create_repl_script(self):
                 with codecs.open(self._user_script, 'rb', 'utf-8') as f:
                     user_script = f.read().rstrip("\r\n") + "\n\n// Frida REPL script:\n"
 
+            if self._codeshare_uri is not None:
```

Comment:
I think it makes sense for `%reload` to reload it tbh, it's what I'd more or less expect it to do, but that just might be me. What do you think? 

> but if we do we need to make sure that it works at that point.

I'm not sure I follow what you're saying here, can you say that a different way? Are you saying that the UI thread could potentially be active by the time this is running?